command/node_status.go

```diff
@@ -347,6 +358,11 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 			basic = append(basic, fmt.Sprintf("Uptime|%s", uptime.String()))
 		}
 
+		// Emit the volume info
+		if !c.verbose {
```

Comment:
It took me a second to understand the logic here.  The comment (here and driver info) isn't informative and misleading.  Maybe more "Emit volume info summary; when in verbose, emit detail info below?" or something like would have helped my reading.

Reply:
Yeah this definitely isn't my favourite code either - I was mostly copying the existing style here, to avoid doing a bigger cleanup. But I guess I'll do that when I come back to this